src/Ordering.API/Application/Commands/SetStockRejectedOrderStatusCommand.cs

```diff
@@ -1,17 +1,3 @@
 ﻿namespace eShop.Ordering.API.Application.Commands;
 
-public class SetStockRejectedOrderStatusCommand : IRequest<bool>
-{
-
-    [DataMember]
```

Comment:
We're losing the `DataMember` attribution on this property. Have we confirmed this doesn't cause any issues?

Reply:
Yes even though we don't use [DataMember] attribute, by using records, the properties are automatically considered for serialization by default.

Added unit test SetStockRejectedOrderStatusCommandTest to test if serialization is working as expected.